ManageWikiSettings.php

```diff
@@ -1187,6 +1187,15 @@
 		'help' => 'This allows to set whether or not it should be possible for anonymous visitors of the wiki to dismiss the site notice shown.',
 		'requires' => [],
 	],
+	'wgDiscordAvatarUrl' => [
+		'name' => 'Discord Notification Avatar URL',
+		'from' => 'mediawiki',
+		'type' => 'text',
```

Comment:
Why not use 'type' => 'url'?

Reply:
Sorry, I was copying wgLogo and didn't realize.

Reply:
That's alright, thanks!